mono/btls/MonoBtls.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,393 @@
+// !$*UTF8*$!
```

Comment:
Looks like you accidentally committed a bunch of xcode project files, please remove them.